src/pages/AddedCard.tsx

```diff
@@ -0,0 +1,175 @@
+import { FC } from 'react';
+import { Box, Button, Container, IconButton, Typography } from '@mui/material';
+import { AppFooter } from '~/shared/ui';
+import { Header } from '~/widgets/header';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import CreateIcon from '@mui/icons-material/Create';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+
+interface AddedCardProps {
+  title: string;
+  cardNumber: string;
+  barcodeNumber: number | string;
+}
+
+export const AddedCard: FC<AddedCardProps> = ({
+  title,
+  cardNumber,
+  barcodeNumber,
+}) => {
+  return (
+    <Container
+      sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}
```

Comment:
Сейчас футер сплющивает, ему нужна вся ширина. Я бы делал stretch и поправил бы горизонтальные паддинги у компонентов. И еще у бокса карты ширину, чтобы тянулась.

```suggestion
      sx={{ display: 'flex', flexDirection: 'column', alignItems: 'stretch' }}
```